samples/19.c.single-sign-on-for-teams-apps/README.md

```diff
@@ -0,0 +1,263 @@
+# Single sign-on demo for Microsoft Teams apps using OAuth
+
+[![Deploy Status](https://fuselabs.vsrm.visualstudio.com/_apis/public/Release/badge/531382a8-71ae-46c8-99eb-9512ccb91a43/12/12)](https://webchat-sample-sso-teams.azurewebsites.net/)
+
+# Description
+
+In this demo, we will show you how to authorize a user to access resources thru a Microsoft Teams app with a bot. We will use [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) for OAuth provider and [Microsoft Graph](https://developer.microsoft.com/en-us/graph/) for the protected resources.
+
+After sign-in, this demo will keep OAuth token inside the Teams tab, and also send it to the bot via Web Chat backchannel. Because both web page and bot need to hold a single OAuth token, we are unable to use OAuth card in this demo.
+
+> When dealing with personal data, please respect user privacy. Follow platform guidelines and post your privacy statement online.
+
+## Background
+
+This sample is a simplified and reduced version of the sample "[Single sign-on demo for enterprise apps using OAuth](https://microsoft.github.io/BotFramework-WebChat/19.a.single-sign-on-for-enterprise-apps)" and modified from "[Single sign-on demo for Intranet apps using OAuth](https://microsoft.github.io/BotFramework-WebChat/19.b.single-sign-on-for-intranet-apps)". There are notable differences:
+
+-  In this demo, we are targeting Microsoft Teams "tab apps", which is a set of web pages browsed thru an embedded and limited web browser inside Microsoft Teams
+   -  Tab apps are supported on desktop client only. Microsoft Teams on mobile client do not support embed content in apps and requires external apps for tab content
```

Comment:
Agree, emphasizing by bold.

Somehow, I don't quite get the difference between emphasizing by italic or bold. Because looks like Markdown prefer italic than bold.